### DISCUSSION TITLE
Adicionar outros campos nas definições como `Fonte` e `Leia mais sobre`.

### DISCUSSION THREAD

**Discussion by george-gca** [levxyca/diciotech#145] [Ideas]
Eu sei que as definições no fundo são criadas a partir de interpretações e simplificações dos próprios contribuidores, então pra dar mais segurança pra quem tá lendo e até pra servir de direcionamento pra quem quiser aprender um pouco mais sobre o termo, acho que seria legal adicionar alguns links no card do termo, como `Fonte` e `Leia mais`.

**Comment by levxyca** [comment]
Oie :) @george-gca! Sobre a questão de adicionar a fonte, conversamos sobre isso nessa issue #86. Mas sobre a ideia de adicionar um "leia mais" eu acho legal, adicionar links de referencias e etc. A gente já tem mapeado ter isso como "materiais complementares", você pode ver mais sobre na issue #12.

**Reply by george-gca** [reply]
Legal, então fechando aqui.
